src/index.js

```diff
@@ -106,7 +108,7 @@ const jsonToFCPX = (sequenceEDLJson)=>{
         const results = composeAsset( {id: `r${counterId}`, 
             src: event.clipName, 
             start: event.startTime, 
-            duration: event.endTime - event.startTime,
+            duration: event.endTime,
```

Comment:
just to clarify this change - the asset element is the whole clip, so duration can just be the end time.

Reply:
So is the start time always 0?

Reply:
ahh yes good point! I will change that too!